.github/workflows/tests.yml

```diff
@@ -672,6 +672,20 @@ jobs:
           export PATH="/home/runner/go/bin:$PATH"
           sudo -E ./lxd-ui/tests/scripts/setup_test
 
+      - name: Test basic LXD functionality
+        shell: bash
+        run: |
+          set -eux
+          export PATH="/home/runner/go/bin:$PATH"
+
+          # launch a test instance
+          sudo -E ./test/deps/import-busybox --alias testimage
```

Comment:
Ah i see what the busybox change was for now.

Reply:
Not exactly as this snippet has a `LXD_DIR` already exported and doesn't use the snap anyway. However it's when I did those steps locally that I figured I could improve the ease of use of this script.